aspnet/fundamentals/dependency-injection.rst

```diff
@@ -1,10 +1,277 @@
-.. _dependency-injection:
+Dependency Injection
```

Comment:
Agree MVC needs its own DI / activation / et cetera article.

This one needs to be more general. It's currently focused on the repository pattern and `Startup`. For example it doesn't mention what any `services.AddBlah()` method is doing. Nor does it mention consuming services outside controller, `Startup`, and repository classes.